data/xml/J79.xml

```diff
@@ -5,335 +5,903 @@
 </paper>
 
 <paper id="1004">
-  <title><i>A Few Steps Towards Computer Lexicometry</i> (Nicholas V Findler; Heino Viil)</title>
+  <title>A Few Steps Towards Computer Lexicometry</title>
+  <author><first>Nicholas V.</first><last>Findler</last></author>
+  <author><first>Heino</first><last>Viil</last></author>
 </paper>
 
 <paper id="1005">
-  <title><i>Pattern-Matching rules for the recognition of Natural Language Dialogue Expressions</i> (Kenneth Mark Colby; Roger C Parkison; Bill Fought)</title>
+  <title>Pattern-Matching Rules for the Recognition of Natural Language Dialogue Expressions</title>
+  <author><first>Kenneth Mark</first><last>Colby</last></author>
+  <author><first>Roger C.</first><last>Parkison</last></author>
+  <author><first>Bill</first><last>Faught</last></author>
 </paper>
 
 <paper id="1006">
-  <title><i>Personal Notes</i>, <i>Computational Semantics Tutorial at Lugano in March</i>, <i>Artificial Intelligence: Directory being compiled</i> (<fixed-case>D</fixed-case>onald E Walker), <i>Letters: Logos Development Corporation on <fixed-case>MT</fixed-case> System</i> (Yorick Wilks), <i>Solar Project Distributes Materials</i> (Tim Diller; John Olney; <fixed-case>N</fixed-case>athan Ucuzoglu), <i><fixed-case>NAS</fixed-case>/<fixed-case>NRC</fixed-case> Studies International Information Programs</i>, <i><fixed-case>NFAIS</fixed-case> Meeting Overlap Study Indexer Training Kit</i> (Ben H Weil), <i>On-line Terminal Searching Course at <fixed-case>P</fixed-case>ratt in <fixed-case>J</fixed-case>anuary</i> (Patricia Breivik), <i>Educational Data Systems Association Convention</i>, <i>Publication Problems: Journal Prices Rising</i> (Philip H Abelson), <i>3rd Pisa Summer School: Report of Courses Lectures</i> (Antonio Zampoli), <i>Summer School at <fixed-case>S</fixed-case>tuttgart: Report of Lectures</i> (Hans Jochen Schneider), <i>Information and Philology Conference Report</i> (Marichal), <i>Ariosto Concordance in Progress</i> (Cesare Segre), <i>Text Data: Round Table on Analytic Procedures Held</i>, <i><fixed-case>SIGLASH</fixed-case></i> (Micheal <fixed-case>L</fixed-case>esk; <fixed-case>R</fixed-case>obert Wachal; Dolores Burton; Karen Mullen), <i>Political Science Concepts to be Collected and Analyzed</i> (<fixed-case>G</fixed-case>eorge J Grahan), <i>Realiable Software Conference Los Angeles <fixed-case>A</fixed-case>pril</i> (M L Shooman), <i>Psycholinguistics Conference New York <fixed-case>J</fixed-case>anuary</i>, <i><fixed-case>NSF</fixed-case>: Excepts from the Organizational Directory</i> (H Guyford Stever), <i>Microfiche Equipment: Background Information for Buyers</i> (Dake Gaddy), <i>Artificial Intelligence in Poland Bibliography</i>, <i><fixed-case>AAAS</fixed-case> Meeting: <fixed-case>J</fixed-case>anuary</i> (W M Carlson), <i>Current Bibliography</i> (Brain <fixed-case>H</fixed-case>arris; R Laskowski)</title>
+  <title>The FINITE STRING, Volume 11, Number 2</title>
```

Comment:
See my comment below (I'm working upwards apparently): I think it would make sense to number this as J79-2XXX.